cost-analyzer/values-unrestricted-podsecurity.yaml

```diff
@@ -1,11 +1,13 @@
 kubecostDeployment:
   podSecurityContext:
-    capabilities:
-      drop:
-      - ALL
-    runAsUser: 1001070000
-    runAsNonRoot: true
-    allowPrivilegeEscalation: false
+    # capabilities:
+    #   drop:
+    #   - ALL
+    fsGroup: 1000980000
+    runAsGroup: 1000980000
+    runAsUser: 1000980000
```

Comment:
@jessegoodier Fyi, in my testing, it is good to use runAsNonRoot: true here without setting up static value for `runAsUser`, `runAsGroup`, `fsGroup` in 4.11